.stylelintrc.json

```diff
@@ -7,6 +7,18 @@
     "no-descending-specificity": null,
     "no-duplicate-selectors": null,
     "number-leading-zero": null,
-    "selector-type-no-unknown": [ true, { "ignore": ["custom-elements"] } ]
+    "selector-type-no-unknown": [ true, { "ignore": ["custom-elements"] } ],
+    "selector-class-pattern": [
+      "^[a-z]+(-[a-z]+)*",
```

Comment:
The package update was failing because the pattern was failing? 

If upstream it is `^([a-z][a-z0-9]*)(-[a-z0-9]+)*$` but here it is `^[a-z]+(-[a-z]+)*`, then isn't the upstream _more_ permissive (allowing digits)?

Reply:
The upstream regex is more limiting as it requires the whole class or ID to match that pattern. Alternatively, we could simply remove the `$` from the pattern and use that (which would then be `^([a-z][a-z0-9]*)(-[a-z0-9]+)*`).

Reply:
Ah, I missed the `$`.

Reply:
Our CSS currently fails the new upstream pattern because of `_` in class names?

Reply:
Once the name deviates from the second capture group (`(-[a-z0-9]+)`), it fails. So that would include patterns such as `_`, `--`, `__` etc.

Reply:
Right. Yeah, I don't think we should enforce that as then we can't use the BEM convention. So your change here is good with me.